scripts/backupDB.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# run this script on YOUR OWN computer
+
+# runs mongodump on the production server
+ssh root@159.203.171.222 "mongodump"
```

Comment:
well, so much for keeping the server ip secret 🔒

Reply:
maybe a bash variable?

Reply:
Yeah, def take this out